test/WormholeL2VoteAggregator.t.sol

```diff
@@ -390,13 +394,43 @@ contract ProposalVoteActive is L2VoteAggregatorTest {
     L2VoteAggregator aggregator =
     new WormholeL2VoteAggregator(address(l2Erc20), L2_CHAIN.wormholeRelayer, address(l2GovernorMetadata), address(l1Block), L2_CHAIN.wormholeChainId, L1_CHAIN.wormholeChainId);
 
-    vm.assume(voteStart > 0); // Underflow because we subtract 1
-    vm.assume(voteStart > block.number); // Block number must
-    vm.assume(voteEnd > aggregator.CAST_VOTE_WINDOW()); // Without we have an underflow
+    vm.assume(voteStart > 0); // Prevent underflow because we subtract 1
+    vm.assume(voteStart > block.number); // Block number must be greater than vote start
+    vm.assume(voteEnd > aggregator.CAST_VOTE_WINDOW()); //  Prevent underflow
     vm.assume(voteEnd - aggregator.CAST_VOTE_WINDOW() > voteStart); // Proposal must have a voting
       // block before the cast
 
-    bytes memory proposalCalldata = abi.encode(proposalId, voteStart, voteEnd);
+    bytes memory proposalCalldata = abi.encode(proposalId, voteStart, voteEnd, isCancelled);
+    vm.prank(L2_CHAIN.wormholeRelayer);
+    l2GovernorMetadata.receiveWormholeMessages(
+      proposalCalldata,
+      new bytes[](0),
+      MOCK_WORMHOLE_SERIALIZED_ADDRESS,
+      L1_CHAIN.wormholeChainId,
+      bytes32("")
+    );
+
+    bool active = aggregator.proposalVoteActive(proposalId);
+    assertFalse(active, "Proposal is supposed to be inactive");
```

Comment:
Should we also have explicit tests to show that casting a vote and bridging a vote will not work when cancelled?

Reply:
https://github.com/ScopeLift/l2-flexible-voting/pull/61/commits/161cca1b8ffe2b0c844c68720ba76ed18c08527d

Reply:
https://github.com/ScopeLift/l2-flexible-voting/pull/61/commits/ebe143e7b1d49e4a6c775727ca9ee4898a214093